src/deleteTask.test.js

```diff
@@ -43,8 +42,8 @@ describe('deleteTask', () => {
 
     deleteTask(taskToDelete, tasks);
 
-    expect(jest.spyOn(indexModule, 'updateTaskIndexes')).toHaveBeenCalled();
-    expect(jest.spyOn(indexModule, 'saveTasks')).toHaveBeenCalled();
+    // expect(jest.spyOn(indexModule, 'updateTaskIndexes')).toHaveBeenCalled();
+    // expect(jest.spyOn(indexModule, 'saveTasks')).toHaveBeenCalled();
```

Comment:
- [ ] Kindly avoid leaving commented pieces of code in your codebase. It makes your codebase unclean and is not a good practice :+1: